composer.json

```diff
@@ -18,10 +18,38 @@
             "wikimedia/composer-merge-plugin": true
         }
     },
+    "repositories": [
+        {
+            "type": "package",
+            "package": {
+                "name": "citation-style-language/locales",
+                "version":"1.0.0",
+                "source": {
+                    "type": "git",
+                    "url": "https://github.com/citation-style-language/locales.git",
+                    "reference": "master"
+                }
+            }
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "citation-style-language/styles-distribution",
```

Comment:
Does anyone know why this repository hasn't updated in almost two years, while the repository it is derived from (https://github.com/citation-style-language/styles) seems to update very frequently? I just want to be sure we're not locking ourselves into something that's outdated/unsupported.

Reply:
Ahh, an answer -- see https://github.com/seboettg/citeproc-php/issues/123 -- I think we need to make updates here.